<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resultados</class>
 <widget class="QWidget" name="Resultados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="main_layout">
   <item>
    <widget class="QGroupBox" name="grupo_gestion_partido">
     <property name="minimumSize">
      <size>
       <width>650</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>650</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Gesti칩n de Partido</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_gestion">
      <item row="10" column="5">
       <widget class="QPushButton" name="btn_anadir_tarjetas">
        <property name="text">
         <string>A침adir</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_jugador_gol">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Jugador</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_goles_registrados">
        <property name="text">
         <string>Goles registrados</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QLabel" name="label_tarjetas_registradas">
        <property name="text">
         <string>Tarjetas registradas</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QComboBox" name="combo_partidos"/>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QComboBox" name="combo_jugador_gol">
        <property name="maximumSize">
         <size>
          <width>218</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_jugador_tarjeta">
        <property name="text">
         <string>Jugador</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="combo_jugador_tarjeta">
        <property name="minimumSize">
         <size>
          <width>174</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="6">
       <widget class="QLabel" name="label_titulo">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Actualizaci칩n de Resultados</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="6">
       <widget class="QTableWidget" name="table_tarjetas">
        <property name="columnCount">
         <number>4</number>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Jugador</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tipo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Minuto</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="btn_anadir_gol">
        <property name="text">
         <string>A침adir</string>
        </property>
       </widget>
      </item>
      <item row="13" column="3">
       <widget class="QPushButton" name="btn_volver">
        <property name="minimumSize">
         <size>
          <width>218</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Volver</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_titulo_goles">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Registrar Goles</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_partido_y_equipos">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Partido y equipos</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QPushButton" name="btn_finalizar_partido">
        <property name="minimumSize">
         <size>
          <width>218</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Partido jugado</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="6">
       <widget class="QTableWidget" name="table_goles">
        <property name="columnCount">
         <number>3</number>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Jugador</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Minuto</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_registrar_tarjetas">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Registrar Tarjetas</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_seleccionar_partido">
        <property name="text">
         <string>Seleccionar partido</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QLabel" name="label_tipo_tarjeta">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Tipo</string>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QComboBox" name="combo_tipo_tarjeta">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Amarilla</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Roja</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_reloj_container" native="true">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
